### PR TITLE
Update schinkenfleckerl-gratinated.md

### PR DESCRIPTION
**Pull request by StohanzlMart**
added cooking time and servings
fixed numbering
fixed https:// prefix

### CODE REVIEW

**Review by DGKSK8LIFE — APPROVED** [review]

File: src/schinkenfleckerl-gratinated.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Schinkenfleckerl überbacken, Viennese Mac'n'Cheese
```
Comment (Knarkzel) [inline src/schinkenfleckerl-gratinated.md:2]: Can you add `.webp` image here?

File: src/schinkenfleckerl-gratinated.md [hunk 2]
```diff
@@ -19,21 +23,19 @@ Serve topped with cheese on top of salad (with Mustard, Jam, Salt, 1 Vinegar, 4
...
 9. Optionally gratin Mozarella or any other cheese on top for 5-10 minutes.
 
 There you have it, an idiot-proof carb and protein bomber that should taste reasonably well. Besides browning the noodles this is really just throwing stuff into a baking pan. This is the kind of dish that tastes better the next day so it's ideal for making on a Sunday and eating for the next 3 or 4 days. Just reheat in oven or pan (add little water).
 
 ## Contribution
 
-- Martin Stohanzl - [website](loesmartin.at), [youtube](https://www.youtube.com/channel/UCJkTyM5-fcVBkhi7DypoNsg)
+- Martin Stohanzl - [website](https://loesmartin.at), [youtube](https://www.youtube.com/channel/UCJkTyM5-fcVBkhi7DypoNsg)
 
 ;tags: austrian beef pork ham noodles smoked-rolled-roast baking-pan
```
Comment (Knarkzel) [inline src/schinkenfleckerl-gratinated.md:41]: 7 tags are too much. Limit these to 3-4 at most.

Comment (DGKSK8LIFE) [reply src/schinkenfleckerl-gratinated.md:41]: 2-3 is probably a better limit for tags

File: src/schinkenfleckerl-gratinated.md (hunk 1 above)
Comment (StohanzlMart) [reply src/schinkenfleckerl-gratinated.md:2]: I'd love to but plan to cook it in a week or two and have no pictures atm.

**Pull request merged**